salt/pkgrepo/ubuntu/init.sls

```diff
@@ -1,3 +1,16 @@
+legacy-saltstack-pkgrepo:
```

Comment:
IMO, people should probably catch this change and not merge it into their fork of the salt-formula until they are ready to upgrade.

IME, people don't use the repos for various reasons.

Hey, you can always add the switch in if someone complains later.